### PR TITLE
Fix (#minor); iron-bank; fixed incorrect APY

### PR DESCRIPTION
**Pull request by tnkrxyz**
Iron-bank actually uses timestamp when calculating borrowing/lending APY on optimism and Avalanche  (see #1746 for the report of the issue and a long discussion), instead of blocks as implemented in the current version of the subgraph. This PR set them to SECONDS_PER_YEAR.

### deployment and validation dashboard

- optimism (sync in progress)
  - deployment:  https://thegraph.com/hosted-service/subgraph/tnkrxyz/iron-bank-optimism?version=pending
  - dashboard: https://subgraphs.messari.io/subgraph?endpoint=https://api.thegraph.com/subgraphs/id/QmQv31JCnuZdw1Q2Ydh9EkSF8iimVZsuX7aTuVYzEN6277&tab=protocol
- avalanche (sync in progress)
  - deployment: https://thegraph.com/hosted-service/subgraph/tnkrxyz/iron-bank-avalanche 
  - dashboard: https://subgraphs.messari.io/subgraph?endpoint=https://api.thegraph.com/subgraphs/name/tnkrxyz/iron-bank-avalanche&tab=protocol

### CODE REVIEW

**Comment by bye43** [comment]
@ishraq8 Can you take a quick look at this when you get the chance?

**Comment by ishraq8** [comment]
Avalanche APY is accurate, Only issue is that pools in Avalanche are labeled as "cream" instead of Iron Bank.

Optimism is also accurate :)

**Comment by tnkrxyz** [comment]
> Avalanche APY is accurate, Only issue is that pools in Avalanche are labeled as "cream" instead of Iron Bank.

That's indeed odd. Looking into it.

File: subgraphs/compound-forks/protocols/iron-bank/src/constants.ts [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
-  AVALANCHE_BLOCKS_PER_YEAR,
   ETHEREUM_BLOCKS_PER_YEAR,
   FANTOM_BLOCKS_PER_YEAR,
   Network,
+  SECONDS_PER_YEAR,
 } from "../../../src/constants";
 
+// See https://github.com/messari/subgraphs/issues/1746 for
+// reason why this is the casef
+const AVALANCHE_BLOCKS_PER_YEAR = SECONDS_PER_YEAR;
```
Comment (dmelotik) [inline subgraphs/compound-forks/protocols/iron-bank/src/constants.ts:11]: I feel like `AVALANCHE_BLOCKS_PER_YEAR` is not the best name for this. We could just use `SECONDS_PER_YEAR` and not create a psuedonym. bc that is not actually blocks/year on avax/optimism

File: subgraphs/compound-forks/protocols/iron-bank/src/constants.ts [hunk 2]
```diff
@@ -1,11 +1,16 @@
 import { Address, dataSource, log, BigInt } from "@graphprotocol/graph-ts";
 import {
-  AVALANCHE_BLOCKS_PER_YEAR,
   ETHEREUM_BLOCKS_PER_YEAR,
   FANTOM_BLOCKS_PER_YEAR,
   Network,
+  SECONDS_PER_YEAR,
 } from "../../../src/constants";
 
+// See https://github.com/messari/subgraphs/issues/1746 for
```
Comment (dmelotik) [inline subgraphs/compound-forks/protocols/iron-bank/src/constants.ts:9]: This is interesting, nice catch by the issue creator, and nice job jumping on it @tnkrxyz

File: subgraphs/compound-forks/protocols/iron-bank/src/constants.ts (hunk 1 above)
Comment (tnkrxyz) [reply subgraphs/compound-forks/protocols/iron-bank/src/constants.ts]: Makes sense. Fixed

**Review by dmelotik — APPROVED** [review]
lgtm

**Pull request merged**